src/components/NewComment.jsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react";
```

Comment:
This file isn't used at all, why is it here? You've added a form to the comments component making this file completely redundant.